samples/cpp/falsecolor.cpp

```diff
@@ -10,24 +13,18 @@ struct ParamColorMar {
     Mat img;
 };
 
-Ptr<Mat> lutRND;
+Mat lutRND(256, 1, CV_8UC3);
 String winName="False color";
 
 static void TrackColorMap(int x, void *r)
 {
-    std::cout << "selected: " << x << std::endl;
     ParamColorMar *p = (ParamColorMar*)r;
     Mat dst;
     p->iColormap= x;
-    if (x == cv::COLORMAP_PARULA + 1)
+    if (x == COLORMAP_PARULA + 1)
     {
-        if (!lutRND)
-        {
-            RNG ra;
-            lutRND = makePtr<Mat>(256, 1, CV_8UC3);
-            ra.fill(*lutRND, RNG::UNIFORM, 0, 256);
-        }
-        applyColorMap(p->img, dst, *lutRND.get());
+        randu(lutRND, Scalar(0, 0, 0), Scalar(255, 255, 255));
```

Comment:
That's the only place where `lutRND` Mat is used. I think `lutRND' declaration should be moved here as well. There is no reason to retain it global since it will anyway be repopulated here on each call.